trie/zk_trie_impl.go

```diff
@@ -534,10 +536,10 @@ func (mt *ZkTrieImpl) GetNode(nodeHash *zkt.Hash) (*Node, error) {
 	mt.lock.RLock()
 	defer mt.lock.RUnlock()
 
-	return mt.getNode(nodeHash)
+	return mt.getNode(nodeHash, nil)
 }
 
-func (mt *ZkTrieImpl) getNode(nodeHash *zkt.Hash) (*Node, error) {
+func (mt *ZkTrieImpl) getNode(nodeHash *zkt.Hash, node *Node) (*Node, error) {
```

Comment:
is this `node` means `nextNode`? if yes, maybe change the name to `nextNode`, because the L546 also have variable named `node`, maybe have shadow variable.

Reply:
Not really, it is just a node instance that caller can provide to avoid allocations.